src/cli/argument-parser.js

```diff
@@ -102,6 +102,7 @@ export default class CLIArgumentParser {
             .option('--app-init-delay <ms>', 'specify how much time it takes for the tested app to initialize')
             .option('--selector-timeout <ms>', 'set the amount of time within which selectors make attempts to obtain a node to be returned')
             .option('--assertion-timeout <ms>', 'set the amount of time within which assertion should pass')
+            .option('--page-load-timeout <ms>', 'set the amount of time within which TestCafe waits for the `window.load` event to fire before starting the test')
```

Comment:
Good question, I guess it works for all page loads in the test. Am I right, @AlexanderMoskovkin ?

Then it should be

```
            .option('--page-load-timeout <ms>', 'set the amount of time within which TestCafe waits for the `window.load` event to fire on page load before proceeding to the next test action')
```